EIPS/eip-4788.md

```diff
@@ -210,20 +224,20 @@ by working backwards from the desired deployment transaction:
   "type": "0x0",
   "nonce": "0x0",
   "to": null,
-  "gas": "0x27eac",
+  "gas": "0x3d090",
   "gasPrice": "0xe8d4a51000",
   "maxPriorityFeePerGas": null,
   "maxFeePerGas": null,
   "value": "0x0",
-  "input": "0x60588060095f395ff33373fffffffffffffffffffffffffffffffffffffffe14604457602036146024575f5ffd5b620180005f350680545f35146037575f5ffd5b6201800001545f5260205ff35b6201800042064281555f359062018000015500",
+  "input": "0x60618060095f395ff33373fffffffffffffffffffffffffffffffffffffffe14604d57602036146024575f5ffd5b5f35801560495762016da0810690815414603c575f5ffd5b62016da001545f5260205ff35b5f5ffd5b62016da042064281555f359062016da0015500",
   "v": "0x1b",
   "r": "0x539",
-  "s": "0x133700f3a77843802897db",
-  "hash": "0x14789a20c0508b81ab7a0287a12a3a41ca960aa18244af8e98689e37ed569f07"
+  "s": "0x133700018971c643803096",
```

Comment:
Do the `r` and `s` values have any particular meaning?  I'm more curious as to why they s value had to change.  Was it searched to find the first valid signature of any address from "zero" or is there some other meaning?

Reply:
The `s` value is like the "nonce" for our address generator. We keep trying new `s` values until we get a deploy address with `beac02` prefix.

Reply:
Got it.  

Is there a canonical definition of what a synthetic address is and best practices to grind out and S value?  

In this particular case (r being 1337 in decimal, and s being 0x1337 left shifted 72 bytes, and then search upwards from there) it is obvious it should not have a known public key.  But I think the industry would benefit from a best practice for showing a transaction is a synthetic address transaction with a know r producing a corresponding s.

Reply:
I think this post is the pinnacle of our knowledge on the subject and it does not give any explicit instructions: https://weka.medium.com/how-to-send-ether-to-11-440-people-187e332566b7

Reply:
Are you talking about [Nothing-up-my-sleeve number](https://en.wikipedia.org/wiki/Nothing-up-my-sleeve_number)s?